### PR TITLE
Adds Stable Mutagen

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
add: Stable Mutagen! Basically a slow acting version of changeling DNA sting
/:cl:

### CODE REVIEW

**Comment by XDTM** [comment]
Should probably give some warning to the host while it's inside

**Comment by MoreRobustThanYou** [comment]
just added that, @XDTM

**Comment by Tlaltecuhtli** [comment]
isnt it the same of mulligan toxin

**Comment by praisenarsie** [comment]
> isnt it the same of mulligan toxin

**Comment by MoreRobustThanYou** [comment]
@Tlaltecuhtli Mulligan toxin randomizes your appearance, Stable Mutagen can turn your appearance into another person's.

**Comment by yourdoom9898** [comment]
Forgive me if I'm wrong, but isn't this basically a chemical version of Envy's Blade?

**Comment by AdamElTablawy** [comment]
It's also a chemical version of the changeling's DNA sting.

On the other hand, who fucking cares?

**Comment by kevinz000** [comment]
transform stings are already chaotic enough, don't need to give chemistry an ability to do this en masse.
:-1:

**Comment by QualityVan** [comment]
Yeah this enables namechange smoke. Which, uh. Not great, on the whole.

**Comment by 81Denton** [comment]
This chem would have a pretty big impact, imho it would be better locked behind uranium/omnizine or as an addition to mulligan/poison kit tator items.

**Comment by Tlaltecuhtli** [comment]
ui ue syringes do this too

**Comment by MoreRobustThanYou** [comment]
Please note: this takes longer than MORPHINE to activate, and also is semi-hard to mass produce (IIRC, acetone requires heating stuff), and will also require draining your own blood... in addition, it gives warnings.

**Comment by ExcessiveUseOfCobblestone** [comment]
acetone requires no heating nor do any of its components.

You are thinking of charcoal which uses oil and requires heating.

**Comment by PKPenguin321** [comment]
Should require large doses to be effective so you can't pump out smoke and have 30 people inhale 0.1u of it and ruin the round

**Comment by MoreRobustThanYou** [comment]
@PKPenguin321 actually, 0.1u would metabolize too quickly for it too work

**Comment by PKPenguin321** [comment]
Still thought I'm thinking like 14u minimum so you need a full syringe of the stuff at the very least

**Comment by MoreRobustThanYou** [comment]
I forgot the formula but there IS a formula for calculating how many units it'd take to reach X amount of cycles

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 1]
```diff
@@ -902,3 +902,36 @@
...
+		for(var/datum/reagent/blood/B in M.reagents.reagent_list)
+			if(B.data && B.data["dna_copy"] && B.data["dna_copy"] != M.dna)
+				dna_to = B.data["dna_copy"]
+	switch(current_cycle)
+		if(1 to 9)
+			if(prob(5))
+				to_chat(M, "<span class='notice'>You feel kinda off.</span>")
+		if(10 to 29)
+			if(prob(15))
+				to_chat(M, "<span class='notice'>You feel like another person.</span>")
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm]: Semantics but perhaps `You don't feel like yourself." would be a bit better?

**Comment by vuonojenmustaturska** [comment]
I think i'd like this more if, instead of putting reaction-specific snowflake into the blood reagent, you had your mutagen + blood combine into the actually active reagent

**Comment by MoreRobustThanYou** [comment]
@vuonojenmustaturska What about the user's OWN blood?

**Comment by KorPhaeron** [comment]
This is bad enough as an antag only thing

**Pull request closed without merging**